plugin/web/styles/components/UI/_icon.less

```diff
@@ -0,0 +1,6 @@
+.StyledSvg  {
+  font-size: 0; 
+  display: inline-block;   
+
```

Comment:
Remove this space. And consider updating the eslint file to include these directories/less files